py2many/clike.py

```diff
@@ -527,6 +527,10 @@ def visit_unsupported_body(self, node, name, body):
                 f"{name} unimplemented on line {node.lineno}:{node.col_offset}"
             )
 
+    def visit_NamedExpr(self, node):
+        target = self.visit(node.target)
```

Comment:
visit node.value as well?

Reply:
I've passed that as the `body` parameter to `visit_unsupported_body`, similar to the other uses.
I expect we'll enhance `visit_unsupported_body` to do that visiting.